app/frontend/users/UserEdit.res

```diff
@@ -337,6 +356,11 @@ let make = (
 
   let (state, send) = React.useReducer(reducer, initialState)
 
+  let cancelNewEmailRequest = () => {
+    Dom.Storage2.removeItem(Dom.Storage2.localStorage, "newEmail")
```

Comment:
This is again a problem associated with using localStorage. We should actually inform the server that the user is cancelling the request to change email. This should clear the new_email column and invalidate the associated tokens.

A new graphQL mutation is required to cancel the email update request.